### ISSUE TITLE
Alarmsirene KEQ1057311

### ISSUE THREAD

**Issue by agsenna** [thkl/hap-homematic#200]
[**Is]( #url) your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
a new device:

KEQ1057311

**Describe alternatives you've considered**

**Additional context**

**Device related data**
If you want to add a new device or add a feature to a new device, please login to your hap-homematic WebUI; Select Internal/Help and create the device description file as described in the dialog you will get. Please attach the file to your request.

**Comment by agsenna** [comment]
{
  "devices": [
    {
      "id": 1000,
      "intf": 0,
      "intfName": "",
      "name": "HM-Sec-SFA-SM",
      "address": "9589912125ABCD",
      "type": "HM-Sec-SFA-SM",
      "channels": [
        {
          "id": 1001,
          "name": "9589912125ABCD:1",
          "intf": 0,
          "address": "9589912125ABCD:1",
          "type": "ALARMACTUATOR",
          "access": 255
        },
        {
          "id": 1002,
          "name": "9589912125ABCD:2",
          "intf": 0,
          "address": "9589912125ABCD:2",
          "type": "ALARMACTUATOR",
          "access": 255
        }
      ]
    }
  ]
}

**Comment by agsenna** [comment]
i am not sure if i did it right, anyway thanks a lot,

**Comment by thkl** [comment]
looks like this is an alarm sounder. what homekit should do ? Make the alarm go off when you press a button ?

**Comment by agsenna** [comment]
that's right, to have the possibility to make in homekit an automation. turn on the alarm and turn it off.

**Comment by agsenna** [comment]
do you think you can get it in homekit?
thanks for your answer.
ps: i was to lay with my answer because the situation...

**Comment by agsenna** [comment]
lieber thomas,

ich schreibe jetzt zurück nach lange zeit.
die covid situation hat mich ein bisschen aufgehalten, aber alles okay.

meinst du kannst du die alarm anlage implementieren.? ich möchte alarm programme über homekit steuern und einstellen.

meinst kannst du das machen.

ich will nämlich nicht die von homematicip außensirine kaufen, wennnich eine habe die 100% noch in ordnung ist. immer die neuste ist nicht gut!

besten dank im voraus und herzliche grüße 
georgios.

Von meinem iPad gesendet
